website/docs/docs/build/metricflow-cli.md

```diff
@@ -0,0 +1,251 @@
+---
+title: MetricFlow CLI
+id: metricflow-cli
+description: "Query metrics and metadata in your dbt project with the metricflow cli"
+sidebar_label: "CLI"
+tags: [Metrics, Semantic Layer]
+---
+
+After you have defined metrics in your dbt project, you can easily query metrics, dimensions, dimension values, and validate your configs using the MetricFlow CLI.
+
+## Installation
+1. Create or activate your virtual environment.`python -m venv venv`
+2. Run `pip install dbt-metricflow`
+
+## CLI commands
+
+## List
+
+This command retrieves metadata values related to [Metrics](/docs/build/metrics-overview), [Dimensions](/docs/build/dimensions), and [Entites](/docs/build/entities) values. 
+
+For example, if you're trying to retrieve the `name` metadata value for a metric, you can use the following command:
+
+
```

Comment:
is that right @Jstein77 ? 

```suggestion

```bash
mf list metrics <metadata value>

```
```

Reply:
No this wouldn't execute

Reply:
ah ok, what's would the correct command for `mf list` be @Jstein77 ?

Reply:
mf list just shows you the available objects you can list by. You would need to run one of the commands below like `mf list metrics` to actually list the values. Running mf list just shows the help text

Reply:
I think we can just add `mf list`

Reply:
oh i gotcha, thanks!